src/hearings/models/linkHearings.model.ts

```diff
@@ -21,6 +30,7 @@ export interface GroupDetailsModel {
 export interface LinkedHearingsDetailModel {
   hearingId: string;
   hearingOrder: number;
+  hearingStage: string;
```

Comment:
this can be removed as it's a BACK END POST model, need to strictly map with it. If it's for showing purpose, it can be transformed from HearingDetailModel.hearingStage.